### PR TITLE
Taxonomies translation

### PR DESCRIPTION
**Pull request by chandonn-zz**
Related to #40 and #65

### CODE REVIEW

File: admin/class-monk-admin.php [hunk 1]
```diff
@@ -627,4 +627,29 @@ public function monk_term_translation_meta_field( $term ) {
 		}
 		require plugin_dir_path( dirname( __FILE__ ) ) . 'admin/partials/admin-monk-term-translation.php';
 	}
+
+	/**
+	 * Function that erases the data stored by the plugin when term is deleted
+	 *
+	 * @param $term object of Term API
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: The `@param` is `$term_id`

File: admin/class-monk-admin.php [hunk 2]
```diff
@@ -627,4 +627,29 @@ public function monk_term_translation_meta_field( $term ) {
...
+
+	/**
+	 * Function that erases the data stored by the plugin when term is deleted
+	 *
+	 * @param $term object of Term API  
+	 *
+	 * @since 1.0.0
+	 */
+	public function monk_delete_term_data( $term_id ) {
+		$monk_term_id           = get_term_meta( $term_id, 'monk_term_id', true );
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Fix `monk_term_id` meta key

File: public/class-monk-public.php [hunk 3]
```diff
@@ -115,4 +115,20 @@ public function enqueue_scripts() {
 		 */
 		wp_enqueue_script( 'monk-language-switcher-script', plugin_dir_url( __FILE__ ) . 'js/monk-widget.js', array( 'jquery', 'jquery-ui-core', 'jquery-ui-selectmenu' ), $this->version, true );
 	}
+
+	public function monk_public_terms_filter( $term_query ) {
+		$args = array();
+
+		if ( ! is_admin() && isset( $_REQUEST['lang'] ) ) {
+			$lang_query = get_query_var( 'lang' );
```
Comment (brenoalvs) [inline public/class-monk-public.php]: `$lang_query` alignment

File: public/class-monk-public.php [hunk 4]
```diff
@@ -115,4 +115,20 @@ public function enqueue_scripts() {
 		 */
 		wp_enqueue_script( 'monk-language-switcher-script', plugin_dir_url( __FILE__ ) . 'js/monk-widget.js', array( 'jquery', 'jquery-ui-core', 'jquery-ui-selectmenu' ), $this->version, true );
 	}
+
+	public function monk_public_terms_filter( $term_query ) {
+		$args = array();
+
+		if ( ! is_admin() && isset( $_REQUEST['lang'] ) ) {
```
Comment (brenoalvs) [inline public/class-monk-public.php]: Use `get_query_var` on isset conditional

**Review by brenoalvs — CHANGES_REQUESTED** [review]
@chandonn I requested some adjustments

File: public/class-monk-public.php [hunk 5]
```diff
@@ -115,4 +115,20 @@ public function enqueue_scripts() {
 		 */
 		wp_enqueue_script( 'monk-language-switcher-script', plugin_dir_url( __FILE__ ) . 'js/monk-widget.js', array( 'jquery', 'jquery-ui-core', 'jquery-ui-selectmenu' ), $this->version, true );
 	}
+
+	public function monk_public_terms_filter( $term_query ) {
```
Comment (leonofre) [inline public/class-monk-public.php:128]: Comments are missing

**Review by leonofre — CHANGES_REQUESTED** [review]

**Review by leonofre — APPROVED** [review]

**Review by brenoalvs — APPROVED** [review]

**Pull request merged**